### PR TITLE
Make kettle, metrics, velodrome unit tests pass on clean bazel install

### PR DESCRIPTION
**Pull request by fejta**
ref https://github.com/kubernetes/test-infra/issues/3037
/assign @cjwagner @apelisse @rmmh

### CODE REVIEW

File: kettle/BUILD [hunk 1]
```diff
@@ -41,6 +41,9 @@ py_test(
     data = ["//:buckets"],
     # idem
     local = True,
+    deps = [
```
Comment (rmmh) [inline kettle/BUILD]: shouldn't this be on oneline for typical bazel style?

Comment (ixdy) [inline kettle/BUILD]: I think both are valid.

**Comment by cjwagner** [comment]
lgtm too

File: kettle/BUILD (hunk 1 above)
Comment (fejta) [inline kettle/BUILD]: If you have only one import then `["foo"]` is valid, otherwise you need to follow this format.

**Pull request merged**